pkg/inventory/manager.go

```diff
@@ -14,28 +14,28 @@ import (
 
 // Manager wraps an Inventory with convenience methods that use ObjMetadata.
 type Manager struct {
-	inventory *actuation.Inventory
+	inventory BaseInventory
```

Comment:
What's the story with this inventory field? Is it a cache? Why does the manager need to cache the inventory?